<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PredictDialog</class>
 <widget class="QDialog" name="PredictDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>起始月份</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DuringMonthBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1个月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2个月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3个月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4个月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5个月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6个月</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>终止月份</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>预测长度</string>
   </property>
  </widget>
  <widget class="QComboBox" name="StartMonthBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>1月</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>1月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6月</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="EndMonthBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>87</width>
     <height>22</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>1月</string>
   </property>
   <item>
    <property name="text">
     <string>1月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6月</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="CountryLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>选择需要查询的国家</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CountryBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>世界</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string extracomment="（包括中国内地、中国香港、中国澳门以及中国台湾省）">中国</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>日本</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>韩国</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>蒙古</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>越南</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>泰国</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>缅甸</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>老挝</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>柬埔寨</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>马来西亚</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新加坡</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>印度尼西亚</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>菲律宾</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>澳大利亚</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>新西兰</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PredictDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PredictDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
